setup.cfg

```
@@ -24,6 +24,7 @@ python_requires = >3.8.0
 install_requires =
     discord.py>=1.5.1
     aiohttp>=3.6.0
+    Red-Commons==1.0.0
```

We can lax it in RLL, especially since we control the Red-Commons library.
```suggestion
    Red-Commons>=1.0.0,<2
```